sql/moz-fx-data-shared-prod/monitoring/missing_namespaces_and_document_types/view.sql

```diff
@@ -0,0 +1,76 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.monitoring.missing_namespaces_and_document_types`
+AS
+WITH error_counts AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    document_namespace,
+    document_type,
+    document_version,
+    error_type,
+    COUNT(*) AS error_count
+  FROM
+    `moz-fx-data-shared-prod.monitoring.payload_bytes_error_structured`
+  WHERE
+    submission_timestamp >= TIMESTAMP_SUB(current_timestamp, INTERVAL 7 DAY)
+    AND exception_class = 'com.mozilla.telemetry.ingestion.core.schema.SchemaNotFoundException'
+  GROUP BY
+    submission_date,
+    document_namespace,
+    document_type,
+    document_version,
+    error_type
+),
+structured_daily_errors AS (
```

Comment:
What's the point of this CTE?

Reply:
mostly just copy-pasted from the Looker view.